### PR TITLE
Pre-v0.2.0 PR: Change package manager to npm, improve readme, update changelog, fix emails/tagging, upgrade package versions

### LINKED ISSUE #93: Clarify usage of yarn vs npm in contributing.md
Right now, it's not clear in `contributing.md` whether yarn or npm should be used. Since `yarn` was chosen to be the package manager, we should more clearly indicate that in contributing.

### PR DESCRIPTION
**Pull request by xhlulu**
## Pre-Merge checklist
- [ ] The project was correctly built with `npm run build:all`.
- [ ] If there was any conflict, it was solved correctly
- [ ] All changes were documented in CHANGELOG.md.
- [ ] All tests on CircleCI have passed.
- [ ] All Percy visual changes have been approved.
- [ ] Two people have :dancer:'d the pull request. You can be one of these people if you are a Dash Cytoscape core contributor.

## Reference Issues

Closes #93 

## Other comments

### CODE REVIEW

**Comment by IvoLeist** [comment]
Hey @xhlulu 

sorry for being so silent in the last weeks but I am right now in a job transition phase
and therefore very busy with preparing a proper handover + doing all the necessary paperwork 
for moving to Barcelona.

Anyway long story short:
I just want to give you a heads up that at least in my old project building with **Yarn**
lead to a buggy behaviour of the compound nodes. 
Meaning if I dragged a compound node I would not move any longer the whole comound node
but each element individually

When using npm everything worked as expected.

Best,
Ivo

**Comment by xhlulu** [comment]
@IvoLeist Thank you however for raising that issue. I noticed that the `react-cytoscapejs` version and the `cytoscape.js` version were outdated (perhaps `yarn` didn't fetch the latest version from `npm` for some reason, but i'm not sure).

**Comment by xhlulu** [comment]
@IvoLeist It seems to work for me
Peek 2020-07-08 12-14

**Comment by IvoLeist** [comment]
I actually never tried the example but directly in our web app (not yet published/ open sourced)
but as soon as you have merged it I can have a try if I still have access to the project 
if not I have just linked that  PR in our project repo

**Comment by xhlulu** [comment]
Amazing! Hope you are (or will be) enjoying Barcelona!

**Comment by xhlulu** [comment]
@IvoLeist After further internal discussions, we think it is indeed be better to use `npm` from now on, for various reasons. I added a few commits to reflect that change. Thank you again for raising this!

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -108,7 +112,7 @@ Those tests are rendered into images located inside `tests/screenshots`, which a
 
 ## Publishing
 
-Create a pull request and tag the Plotly team (`@plotly/dash_bio`) and tag / request review from [@xhlulu](https://github.com/xhlulu).
+Create a pull request and tag the Plotly team (`@plotly/dash`) and tag / request review from [@xhlulu](https://github.com/xhlulu).
```
Comment (alexcjohnson) [inline CONTRIBUTING.md:115]:
```suggestion
Create a pull request and tag the Plotly team (`@plotly/dash-core`) and tag / request review from [@xhlulu](https://github.com/xhlulu).
```

Even better though would be to do this with a `CODEOWNERS` file.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -190,12 +194,7 @@ Complete the "Publishing" section.
 - [ ] Step 1 and 2 of Post-merge checklist are completed.
 - [ ] You have closed all issues that this pull request solves, and commented the new version number users should install.
 - [ ] If significant enough, you have created an issue about documenting the new feature or change and you have added it to the [Documentation] project.
-- [ ] You have created a pull request in [Dash Docs] with the new release of your feature by editing that project's [`requirements.txt` file](https://github.com/plotly/dash-docs/blob/master/requirements.txt) and you have assigned `@chriddyp` to review.
-
-## Notes
-
-#### Package manager
-Our preferred package manager for this project is Yarn. Therefore we use `yarn.lock` rather than `package-lock.json`. If you decide to start using npm for package management (which will create package-lock.json) and you commit this project to Dokku, make sure to delete `yarn.lock`.
+- [ ] You have created a pull request in [Dash Docs] with the new release of your feature by editing that project's [`requirements.txt` file](https://github.com/plotly/dash-docs/blob/master/requirements.txt) and you have assigned `@plotly/dash` to review.
```
Comment (alexcjohnson) [inline CONTRIBUTING.md:197]:
```suggestion
- [ ] You have created a pull request in [Dash Docs] with the new release of your feature by editing that project's [`requirements.txt` file](https://github.com/plotly/dash-docs/blob/master/requirements.txt).
```

`dash-docs` does this via [`CODEOWNERS`](https://github.com/plotly/dash-docs/blob/master/.github/CODEOWNERS)

**Comment by xhlulu** [comment]
@alexcjohnson I created a CODEOWNERS file. Is there any way to see if it shows the expected results?

File: .github/CODEOWNERS [hunk 3]
```diff
@@ -0,0 +1,3 @@
+# These owners will be the default owners for everything in
+# the repo. Unless a later match takes precedence
+* @alexcjohnson @Marc-Andre-Rivet @xhlulu @chriddyp @plotly/dash-core
```
Comment (alexcjohnson) [inline .github/CODEOWNERS:3]:
```suggestion
* @alexcjohnson @Marc-Andre-Rivet @xhlulu
```

Let's go with the same model dash-bio has - reducing noise for Chris and others in `dash-core`.

I don't think we need to test it, it looks fine - next time someone makes a PR we'll see it in action.

**Pull request merged**